po/glossary/nl.po

```diff
@@ -477,10 +477,8 @@ msgid "loan"
 msgstr "lening"
 
 #. "Annual Percentage Rate, includes in contrast to the Nominal Rate some of the various costs to obtain a loan."
-#, fuzzy
-#| msgid "loan"
 msgid "loan: APR"
-msgstr "lening"
+msgstr "lening: APR"
```

Comment:
Most [Belgian](http://www.aflossingstabel.be/bereken-aflossing.html) [online](http://www.aflossingstabel.be/) [loan](https://financeinfo.be/leningen/aflossingstabel/)  calculators I have found use the term `JKP` (Jaarlijks Kosten Percentage) rather than APR.